WooCommerce/Classes/ViewRelated/Dashboard/Settings/CardReadersV2/CardReaderSettingsKnownViewModel.swift

```diff
@@ -0,0 +1,228 @@
+import Foundation
+import Combine
+import Yosemite
+
+enum CardReaderSettingsKnownViewModelState {
+    case idle
+    case searching
+    case stoppingSearch
+    case searchFailure(Error)
+    case foundUnknownReader
+    case connectingToReader
+    case connectionFailure(Error)
+    case connected
+}
+
+final class CardReaderSettingsKnownViewModel: CardReaderSettingsPresentedViewModel {
+
+    private(set) var shouldShow: CardReaderSettingsTriState = .isUnknown
+    var didChangeShouldShow: ((CardReaderSettingsTriState) -> Void)?
+    var didUpdate: (() -> Void)?
+
+    private var noConnectedReaders: CardReaderSettingsTriState = .isUnknown
+    private var noKnownReaders: CardReaderSettingsTriState = .isUnknown
+    private var knownReadersProvider: CardReaderSettingsKnownReadersProvider?
+    private var siteID: Int64 = Int64.min
```

Comment:
It seems like the siteID is assigned in the initializer and never modified? If that's the case, it could be a let instead of a var

Reply:
Fixed in https://github.com/woocommerce/woocommerce-ios/pull/4565/commits/82ca59bdac562dc1a7f3408ff317d37f20fbabfc